### PR TITLE
Added libraries needed to build an AMI for discovery deployment

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

**Comment by jibsheet** [comment]
Assuming CI passes, this looks fine 👍

**Comment by clintonb** [comment]
@edx/devops please review
FYI @edx/ecommerce

File: playbooks/roles/discovery/defaults/main.yml [hunk 1]
```diff
@@ -199,5 +199,7 @@ discovery_debian_pkgs:
   - libssl-dev
   - libffi-dev  # Needed to install the Python cryptography library for asymmetric JWT signing
   - libmemcached-dev # Needed for pylibmc
+  - libxml2-dev
+  - libxslt-dev
```
Comment (rlucioni) [inline playbooks/roles/discovery/defaults/main.yml:203]: @schenedx is going to need libjpeg-dev very soon with his image work. Adding it as part of this PR would speed things up for us there.

Comment (clintonb) [inline playbooks/roles/discovery/defaults/main.yml:203]: Updated.

**Comment by rlucioni** [comment]
👍

**Comment by schenedx** [comment]
👍

**Pull request merged**